sky/resources.py

```diff
@@ -332,6 +319,21 @@ def _try_validate_accelerators(self) -> None:
             # because e.g., the instance may have 4 GPUs, while the task
             # specifies to use 1 GPU.
 
+        # Validate whether accelerator is available in specified region/zone.
+        if self.accelerators is not None:
+            acc, _ = list(self.accelerators.items())[0]
+            if self.region is not None or self.zone is not None:
+                if not self._cloud.accelerator_in_region_or_zone(
```

Comment:
Do we want to test the number of accelerators as well? Is it possible that some region/zones may have different number of accelerators, like 8 V100 only appears in some regions?

Reply:
Just added the code to check `acc_count` in the catalog.